src/radembedded.c

```diff
@@ -11,167 +11,194 @@
 
 #define AUTH_PORT_PASSWORD ":1812:testing123"
 #define ACCT_PORT_PASSWORD ":1813:testing123"
-#define SERVER_ADDR "localhost"
+#define SERVER_ADDR        "localhost"
 
-int
-main (int argc, char **argv)
+int main(int argc, char** argv)
 {
-	int		retval = 0;
-	rc_handle 	*rh = NULL;
-	uint32_t 	client_port = 0;
-	uint32_t	status_type = PW_STATUS_STOP;
-        VALUE_PAIR      *send = NULL;
+    int         retval              = 0;
```

Comment:
You change the formatting on this file making the changes unreadable. Please use the original format.